src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1787,7 +1769,6 @@ because xbuild doesn't support framework reference assemblies.
     UncompressedFileExtensions="$(AndroidStoreUncompressedFileExtensions)"
     AssetDirectory="$(MonoAndroidAssetsDirIntermediate)"
     ToolPath="$(AaptToolPath)"
-    ToolExe="$(AaptToolExe)"
```

Comment:
We don't want to remove this property, which means we'll need to add a `ToolExe` property and corresponding logic.

The rationale is that if (when?) Google changes the location of `aapt` in the future, we'd like developers to be able to specify the new location (file name?) without requiring a new release from us, Overriding `$(AaptToolPath)` and `$(AaptToolExe)` are the conventional ways to do this, which in turn means we need to continue supporting them.

(...and we should document those. doh!)